chunky/src/java/se/llbit/tiff/TiffFileWriter.java

```diff
@@ -233,7 +237,8 @@ public void write32(Scene scene, TaskTracker.Task task) throws IOException {
       task.update(height, y);
       for (int x = 0; x < width; ++x) {
         double[] pixel = new double[3];
-        scene.postProcessPixel(x, y, pixel);
+        ((PixelPostProcessingFilter) filter)
+            .processPixel(width, height, scene.getSampleBuffer(), x, y, scene.getExposure(), pixel);
```

Comment:
Maybe remove the Scene.postProcessPixel function as I think this was the only call site